spec/wor/test_services/some_service.rb

```diff
@@ -0,0 +1,77 @@
+require 'wor/requests/base'
+require 'wor/requests/request_error'
+require 'wor/requests/version'
+require 'logger'
+
+class SomeService < Wor::Requests::Base
+  def get_with_log_without_block
+    get(
+      attempting_to: 'attempting_to get mypath',
+      path: '/mypath'
+    )
+  rescue Wor::Requests::RequestError => e
+    e
+  end
+
+  def get_with_log_with_block
+    get(
+      attempting_to: 'attempting_to get mypath',
+      path: '/mypath'
+    ) { |response| JSON.parse(response.body)['array'].reduce('', :+) }
+  rescue Wor::Requests::RequestError => e
+    e
+  end
+
+  def get_without_log_without_block
+    get(
+      path: '/mypath'
+    )
+  rescue Wor::Requests::RequestError => e
+    e
+  end
+
+  def get_without_log_with_block
+    get(
+      path: '/mypath'
+    ) { |response| JSON.parse(response.body)['array'].reduce('', :+) }
+  rescue Wor::Requests::RequestError => e
+    e
+  end
+
+  def get_without_rescue
+    get(
+      path: '/mypath'
+    )
+  end
+
+  VALID_HTTP_VERBS = %i(get post patch put delete).freeze
```

Comment:
You can use the array defined in Wor::Requests::Base::VALID_HTTP_VERBS. I think this line is not necessary